frontend-cert/css-projects/css-grid/user-stories.md

```diff
@@ -0,0 +1,20 @@
+1. You should have a `div` element with the class `magazine-cover`.
+1. Your `.magazine-cover` should have five `div` elements nested inside it.
+1. Each nested `div` should have the classes `title`, `feature-article`, `small-article1`, `small-article2`, and `cover-image`.
+1. The `.magazine-cover` should have a display property of `grid`.
+1. The `.magazine-cover` should have a `grid-template-areas` property of:
+   ```
+   "title title title"
+   "feature-article feature-article cover-image"
+   "small-article1 small-article2 cover-image"
+   ```
+1. The `.magazine-cover` should have a `grid-template-columns` property divided into three parts.
+1. The `.magazine-cover` should have a `grid-template-rows` property of auto for the first row and equal heights for the remaining two rows.
+1. Add a `gap` of `10px` and a `width` of `80%` to the `.magazine-cover`.
+1. Set the background color of the `.magazine-cover` to `#fff`.
```

Comment:
I think we could get rid of this user story. It doesn't really have anything to do with grid - users can set or not set the color if they want.